biojava-structure/src/main/java/org/biojava/nbio/structure/SubstructureIdentifier.java

```diff
@@ -316,4 +321,97 @@ public Structure loadStructure(AtomCache cache) throws IOException, StructureExc
 		return cache.getStructureForPdbId(pdb);
 	}
 
+	/**
+	 * Supplements the reduced structure with ligands from the full structure based on
+	 * a distance cutoff. Ligand groups are moved (destructively) from full to reduced
+	 * if they fall within the cutoff of any atom in the reduced structure.
+	 * The {@link StructureTools#DEFAULT_LIGAND_PROXIMITY_CUTOFF default cutoff}
+	 * (currently 7Å) is used.
```

Comment:
It was changed to 5A. Maybe it is better no to specify in the javadoc what the cutoff value is.